internal/speedtest/config/config.go

```
@@ -0,0 +1,264 @@
+	num, err := strconv.ParseFloat(m[1], 64)
+	if err != nil {
+		return 0, err
+	}
+	if num < 0 {
+		return 0, fmt.Errorf("size must be non-negative")
+	}
+	unit := strings.ToLower(m[2])
+	if unit == "" {
+		return int64(num), nil
+	}
+	mul := int64(1)
```

golangci-lint (2.11.4)

[error] ineffectual assignment to mul

(ineffassign)